source/gui/settingsDialogs.py

```diff
@@ -3129,6 +3144,9 @@ def onOpenScratchpadDir(self,evt):
 		path=config.getScratchpadDir(ensureExists=True)
 		os.startfile(path)
 
+	def onWasapiChange(self, evt):

```

Comment:
```suggestion
	def onWasapiChange(self, evt: wx.SpinEvent):
```

Reply:
I'm fairly sure checkbox events are CommandEvent, not SpinEvent, so I went with CommandEvent.